analyses/focal-cn-file-preparation/06-find-recurrent-calls.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Determine the recurrent focal CN dominant status calls"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Chante Bethell for ALSF CCDL
+date: 2020
+---
+
+This notebook determines the recurrent focal copy number dominant status calls by region using the output of `05-define-most-focal-cn-units.Rmd`.
+
+Recurrence here has been arbitrarily defined based on the plotting of the distribution of status calls and a [similar decision](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/d197f2717eb0d067fe35644db91c4910b6d# 46c06/analyses/fusion_filtering/06-recurrent-fusions-per-histology.R#L111) made in `analyses/fusion_filtering/06-recurrent-fusions-per-histology.R`
```

Comment:
This link isn't working for me.

Reply:
Is the similar decision you are referring to here the distribution plot or something else? If it's something else can you describe this here?

Reply:
Can you add a bit more rationale about how the cutoff has come to be? You may also want to add a a thing in the set up where we can easily change the cutoff if needed.

Reply:
The similar decision made here is the cutoff. I can fix the link and add a better description here.

Reply:
I updated this description in the last commit and in the rendered notebook [here](https://cbethell.github.io/open-pbta-output/06-find-recurrent-calls.nb.html). Let me know if this is along the lines of what you were looking for!